parity-util-mem/Cargo.toml

```diff
@@ -7,6 +7,10 @@ description = "Collection of memory related utilities"
 license = "MIT/Apache-2.0"
 edition = "2018"
 
+# Prevent multiple versions from being linked into the same program.
```

Comment:
> maybe I would add to parity-util-mem/Cargo.toml comment that it is a bit of a hack (so if it breaks at some point we will not look to much).

you mean here? FWIW ring has been using it for a long time https://github.com/briansmith/ring/blob/ddfa683266e8ce41c06feb9f46503dd4e1d47ab1/Cargo.toml#L15-L16, so I don't expect it to break anytime soon

Reply:
Yes I think it is ok (the empty build.rs can be surprising at first).

Reply:
I added it only because `links` is not allowed otherwise (which makes sense).

Reply:
>  added it only because `links` is not allowed otherwise

Let's add a comment either here or in `build.rs` to explain this? A link to https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key might be enough.